cmd/lncli/commands_test.go

```diff
@@ -68,3 +71,64 @@ func TestParseChanPoint(t *testing.T) {
 		}
 	}
 }
+
+// TestParseTimeLockDelta tests parseTimeLockDelta with various
+// valid and invalid input values and verifies the output.
+func TestParseTimeLockDelta(t *testing.T) {
+	t.Parallel()
+	testCases := []struct {
+		timeLockDeltaStr      string
+		expectedTimeLockDelta uint16
+		expectErr             bool
+		expectedErrContent    string
+	}{
+		{
+			"-1",
+			0,
+			true,
+			fmt.Sprintf("fail to parse time_lock_delta: %d "+
+				"to uint64", -1),
+		},
+		{
+			"0",
+			0,
+			false,
+			"",
+		},
+		{
+			"3",
+			3,
+			false,
+			"",
+		},
+		{
+			"65535",
+			65535,
```

Comment:
Why not re-use `math.MaxUint16` here?

Reply:
Also for uint64, you can use something like this
```
strconv.FormatUint(math.MaxUint64, 10)
```

this way it's more clear that one case checks the uint64 limits